rostwitter/python/rostwitter/twitter.py

```diff
@@ -73,19 +73,40 @@ def _check_and_split_word(self, text):
                 text = ''
         return text
 
+    def _post_update_with_reply(self, texts, in_reply_to_status_id=None):
```

Comment:
did you add some feature more than 'enablt to tweet with 280 characters????`

Reply:
> Also, this PR enables thread tweet by using reply.
> Changed so that strings longer than 140 characters are displayed as a thread.
> https://twitter.com/pr2jsk/status/1559451846418530305

As explained at the beginning of this PR, the REPLY function is used to change sentences that exceed 140 characters to be displayed THREAD.

Reply:
ok, please update your example tweet --- Real exampe, generated by working demo program, not a test code. Good picture, that make people to understand what the robot is doing, why the need to tweet more than 280 characters.